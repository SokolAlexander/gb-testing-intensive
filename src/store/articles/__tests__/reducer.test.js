import reducer from "../reducer";
import { getArticlesRequest } from "../actions";

describe('articles reducer tests', () => {
  it("matches snapshot after getArticlesRequest", () => {
    const result = reducer(undefined, getArticlesRequest());

    expect(result).toMatchSnapshot();
  });
});
