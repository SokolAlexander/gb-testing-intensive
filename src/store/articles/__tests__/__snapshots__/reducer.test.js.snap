// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`articles reducer tests matches snapshot after getArticlesRequest 1`] = `
Object {
  "articles": Array [],
  "request": Object {
    "error": null,
    "status": 1,
  },
}
`;
